src/Main.java

```diff
@@ -1,5 +1,73 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        //Задача 1
+
+        byte age = 15;
+
+        if (age >=18) {System.out.println("Если возраст человека равен "+age+", то он совершеннолетний");}
+        else {System.out.println("Если возраст человека равен "+age+", то он не достиг совершеннолетия, нужно немного подождать");}
+
+        //Задача 2
+
+        byte temp=5;
+        boolean weatherCondition= temp>=5;
+
+        if (weatherCondition) {System.out.println("Сегодня тепло, можно идти без шапки");}
+        else {System.out.println("На улице холодно, нужно надеть шапку");}
+
+        //Задача 3
+
+        short speed=59;
+
+        if (speed>=60) {System.out.println("Если скорость "+speed+" то придется заплатить штраф");}
+        else {System.out.println("Если скорость "+speed+" то можно ездить спокойно");}
+
+        //Задача 4
+
+        byte age4=60;
```

Comment:
зачем заводить новую переменную, если можно просто переиспользовать уже объявленную?
```suggestion
        age = 60;
```

Reply:
Открываем задачу и смотрим критерии- "ИНИЦИАЛИЗИРОВАЛИ переменные", логично что 4 соответствует номеру задачи.

Reply:
да, именно, что ИНИЦИАЛИЗИРОВАЛИ, а не СОЗДАЛИ/ОБЪЯВИЛИ.

Reply:
> логично что 4 соответствует номеру задачи

Логично, но создавать 100500 переменных очень плохая практика, лучше сразу учиться писать код правильно, а не как попало.

Reply:
>  а не как попало.
А где ты видишь "как попало?" Задача с отдельным номером, и я сам не хотел бы объявлять ее, но у меня возникла мысль, что будет замечание о том, что прошлая переменная принадлежит строго первой задаче и что везде должен быть порядок, и в отдельной задаче должны быть свои переменные, что надо учиться код писать правильно. 
Ты точно занимаешься тем, чем должен? Ты точно щас должен мне доказать, что я не прав?

Reply:
> Ты точно занимаешься тем, чем должен?

За 2 года и несколько сотен студентов ты первый, кто мне задаёт этот вопрос. Что ж, думаю, что да, тем, чем должен, раз вижу результат своих трудов - трудоустроенных студентов после обучения.

> Ты точно щас должен мне доказать, что я не прав?

Честно, мне пофиг :) Я просто делаю свою работу - учу и передаю опыт, как надо правильно писать код. Дальше уже дело каждого конкретного студента - прислушаться или делать как сам знаешь (тогда вопрос - зачем тогда пришёл учиться, если сам знаешь, как "правильно"?)